modules/cats-effect/README.md

```diff
@@ -23,11 +23,13 @@ To load a configuration file from a path using cats-effect's `IO`:
 import pureconfig._
 import pureconfig.generic.auto._
 import pureconfig.module.catseffect.syntax._
-import cats.effect.IO
+import cats.effect.{ Blocker, IO }
 
-case class MyConfig(somefield: Int, anotherfield: String)
+final case class MyConfig(somefield: Int, anotherfield: String)
```

Comment:
Is this `final` necessary to make `Blocker` work? If not, could you leave it as it was before?

Reply:
Not really, is just that case classes have to be final _(since extending them is a super bad practice)_ I thought it was minor change, but if you prefer to leave it as it was before, no problem. It was just my ocd :sweating_smile: